tools/topology/m4/pcm.m4

```diff
@@ -74,6 +74,20 @@ define(`PCM_CAPABILITIES',
 dnl PCM_PLAYBACK_ADD(name, pcm_id, playback)
 define(`PCM_PLAYBACK_ADD',
 `ifelse(`$#', `3',
+PCM_PLAYBACK_ADD_COMMON($1, $2, $3, false),
+`fatal_error(`Invalid parameters ($#) to PCM_PLAYBACK_ADD')')'
+)
+
+dnl PCM_PLAYBACK_COMPR_ADD(name, pcm_id, playback)
+define(`PCM_PLAYBACK_COMPR_ADD',
```

Comment:
COMPR_PLAYBACK_ADD?

This would be directly visible as a compressed pipeline.

Reply:
I don't think I understand the question :D.

Reply:
I was suggesting a change from PCM_PLAYBACK_COMPR_ADD to COMPR_PLAYBACK_ADD.

Reply:
Oh, yes I also thought about the name but it is not very clear for me the distinction between PCM and Compress, since "compress" is just a property of the PCM in the topology file. 

Anyway COMPR_PLAYBACK_ADD sounds much better, will update the PR.

Reply:
@plbossart done.